src/util/kaldi-thread-test.cc

```diff
@@ -128,6 +128,6 @@ void TestTaskSequencer() {
 int main() {
   using namespace kaldi;
   TestThreads();
-  for (int32 i = 0; i < 1000; i++)
+  for (int32 i = 0; i < 10; i++)
```

Comment:
Why decrease this?

Reply:
It seems done by Dan.

Reply:
The reason I reduced the number of loops in TestThreads(), is that on some machines, with the latest CuDevice code that uses thread_local variables, if too many threads are launched too quickly it uses up some kind of system resource and then fails to allocate a thread.  I forget the exact error message.

Reply:
Okay